src/utils/apiStructure.js

```diff
@@ -1,5 +1,22 @@
 export default {
-  comics: 'getComics',
-  characters: 'getCharacters',
-  all: 'getRandomCharacters'
+  comics: {
+    search: 'searchComics',
+    get: 'getComicsById'
+  },
+  characters: {
+    search: 'searchCharacters',
+    get: 'getCharacterById'
+  },
+  stories: {
+    get: 'getStoryById'
+  },
+  series: {
+    get: 'getSeriesById'
+  },
+  creators: {
+    get: 'getCreatorById'
+  },
+  all: {
+    search: 'getRandomCharacters'
+  }
```

Comment:
Such a nice structure! 👍 
Very informative! 😄

Reply:
Is it a joke?)

Reply:
> Such a nice structure! 👍
> Very informative! 😄

Is it a joke?)

Reply:
Probably I should rename "get" to "getById"

Reply:
1. Nope, I was really glad to see a kind of `model` structure here. Most time this is a self documented peace of code which helps to understand how app is working and what it's doing
2. yep, `getById` would be more informative 👍